install.sh

```diff
@@ -177,6 +177,19 @@ nvm_check_global_modules() {
   fi
 }
 
+nvm_bash_completion() {
+  NVM_PROFILE=$(nvm_detect_profile)
+  if [ -z "$NVM_PROFILE" ] ; then
```

Comment:
Oh, I do! But in the event that it returns a profile file that can't handle `bash_completion`, perhaps it should be a noop?